tool.ts

```diff
@@ -260,14 +279,46 @@ export async function downloadTool(
             file.on('error', (err) => {
                 file.end();
                 reject(err);
-            })
+            });
         }
         catch (error) {
             reject(error);
         }
     });
 }
 
+//---------------------
+// Size functions
+//---------------------
+
+/**
+ * Gets size of downloaded file from "Content-Length" header
+ *
+ * @param response    response for request to get the file
+ */
+function _getContentLengthOfDownloadedFile(response: httpm.HttpClientResponse): number {
+    let contentLengthHeader = response.message.headers['content-length']
+    let parsedContentLength = parseInt(contentLengthHeader);
+    return parsedContentLength;
+}
+
+/**
+ * Gets size of file saved to disk
+ *
+ * @param filePath    the path to the file, saved to the disk
+ */
+function _getFileSizeOnDisk(filePath: string): number {
+    try {
+        let fileStats = fs.statSync(filePath);
+        let fileSizeInBytes = fileStats.size;
+        return fileSizeInBytes;
+    }
+    catch (err) {
+        tl.warning(`Unable to find file size for ${filePath} due to error: ${err.Message}`);
```

Comment:
Nit: could it make sense to move it out of this function - to stay align with SOLID?
If someone will want to use _getFileSizeOnDisk somewhere else in task, but in silent mode - they would need to refactor